CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ Replacement `tsi1` indexes will be automatically generated on startup for shards
 1. [20565](https://github.com/influxdata/influxdb/pull/20565): Update V1 API spec to document all valid Accept headers and matching Content-Types.
 1. [20578](https://github.com/influxdata/influxdb/pull/20578): Respect the --skip-verify flag when running `influx query`.
 1. [20495](https://github.com/influxdata/influxdb/pull/20495): Update Flux functions list in UI to reflect that `v1` package was renamed to `schema`.
+1. [19764](https://github.com/influxdata/influxdb/pull/20620): Add shard duration settings to CLI and API
```

Comment:
After rebasing onto / merging with latest `master`, this will need to move up to the "Features" section under "unreleased". Would propose:
```suggestion
1. [20620](https://github.com/influxdata/influxdb/pull/20620): Add support for explicitly setting shard-group durations. Thanks @hinst!
```